app.js

```diff
@@ -144,12 +144,13 @@ var config = {
  *
  * @memberof! app
  */
-var logger = new Logger({
-	echo: process.env.LOG_LEVEL || appConfig.consoleLogLevel,
-	errorLevel: process.env.FILE_LOG_LEVEL || appConfig.fileLogLevel,
+var logger = createLogger({
+	level: process.env.FILE_LOG_LEVEL || appConfig.fileLogLevel,
 	filename: appConfig.logFileName,
 });
```

Comment:
can we add `createLogger` specific name parameter here like `name: core` `name: p2p` so on and so forth, and in the `bunyan.createLogger` functions instead of using filename for the name parameter u can use the custom name `name: core` `name: p2p` we passed while creating a logger, the log would look much clear and better
From:
```
{"name":"logs/lisk.log","hostname":"MacBook-Pro-6.local","pid":48427,"module":"delegates"}
```
To:
```
{"name":"core","hostname":"MacBook-Pro-6.local","pid":48427,"module":"delegates"}
```
```
{"name":"database","hostname":"MacBook-Pro-6.local","pid":48427,"module":"delegates"}
```